### PR TITLE
Create popup info view

### PR DESCRIPTION
**Pull request by iOSKirill**
1. Added popup info view. 
2. There is a problem when opening popup info, the blur is not superimposed on the button that is located in the NavigationBar.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
HeartRateMonitorApp_—_CustomPopupVStackInfo_swift
please fix warnings

**Review by ghost — COMMENTED** [review]
HeartRateMonitorApp_—_TabBarView_swift

**Review by ghost — COMMENTED** [review]
iPhone_15_Pro_and_Heart_Rate_Monitor___Pulse_App_and_HeartRateMonitorApp_—_TabBarView_swift

**Review by ghost — COMMENTED** [review]
simulator_screenshot_5CD7F093-4352-4E5C-8745-04EC614E1566

File: HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift [hunk 1]
```diff
@@ -226,12 +277,17 @@ struct HomeHealthView: View {
                         weeklyAssessmentDashboard
                     }
                 }
+                if viewModel.isPopupVisible {
+                    Color.black.opacity(0.6)
+                        .edgesIgnoringSafeArea(.all)
+                    popupInfo
+                }
```
Comment (ghost) [inline HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: please present popup with animation

File: HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift [hunk 2]
```diff
@@ -60,6 +60,57 @@ struct HomeHealthView: View {
...
+                CustomPopupVStackInfo(
+                    image: .popupHRVIcon,
+                    title: L10n.Popup.Hrv.title,
+                    subtitle: L10n.Popup.Hrv.subtitle
+                )
+                Button {
+                    viewModel.isPopupVisible.toggle()
+                } label: {
+                    VStack {
+                        Text("OK")
```
Comment (ghost) [inline HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: please localize

**Review by ghost — CHANGES_REQUESTED** [review]

File: HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift (hunk 1 above)
Comment (iOSKirill) [reply HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: Fixed

File: HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift (hunk 2 above)
Comment (iOSKirill) [reply HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: Fixed

File: HeartRateMonitorApp/HeartRateMonitorApp/CustomViews/CustomNavigationBar.swift [hunk 3]
```diff
@@ -0,0 +1,56 @@
+//
+//  CustomNavigationBar.swift
+//  HeartRateMonitorApp
+//
+//  Created by Kirill Manuilenko on 12.12.23.
+//
+
+import SwiftUI
+
+struct ScrollPreKey: PreferenceKey {
```
Comment (ghost) [inline HeartRateMonitorApp/HeartRateMonitorApp/CustomViews/CustomNavigationBar.swift]: ScrollPreferenceKey will be better name

File: HeartRateMonitorApp/HeartRateMonitorApp/CustomViews/CustomNavigationBar.swift [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    static var defaultValue: CGFloat = 0
+    static func reduce(value: inout CGFloat, nextValue: () -> CGFloat) {
+        value = nextValue()
+    }
+}
+
+struct CustomNavigationBar: View {
+    // MARK: - Property -
+    @StateObject var viewModel = HomeHealthViewModel()
+    @Binding var isScroll: Bool
```
Comment (ghost) [inline HeartRateMonitorApp/HeartRateMonitorApp/CustomViews/CustomNavigationBar.swift]: what does it mean? is scroll what? is scrolling? or is it scroll view?

File: HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift [hunk 5]
```diff
@@ -61,16 +45,70 @@ struct HomeHealthView: View {
...
         VStack(alignment: .leading) {
             HStack {
                 Text(L10n.Dashboard.Measure.title)
-                    .font(.custom(FontFamily.Urbanist.bold, size: 17))
+                    .font(.appUrbanistBold(of: 17))
                     .foregroundColor(Color.white)
                 Spacer()
                 Button {
-                    // Additional information
+                    withAnimation(.linear(duration: 0.5)) {
```
Comment (ghost) [inline HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: oh, it looks too slow
try this one, looks much better from my point of view 
`withAnimation(.easeInOut(duration: 0.3)) {`

Comment (ghost) [inline HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: dismiss also should be with animation

File: HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift [hunk 6]
```diff
@@ -213,26 +251,36 @@ struct HomeHealthView: View {
...
+                    .frame(height: 30)
+            })
+            .overlay {
+                CustomNavigationBar(isScroll: $viewModel.isScroll)
+            }
+            if viewModel.isPopupVisible {
+                Color.black.opacity(0.6)
+                    .edgesIgnoringSafeArea(.all)
+                popupInfo
+                    .statusBar(hidden: true)
```
Comment (ghost) [inline HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: don't hide the status bar, the content jumps up, it looks weird

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
simulator_screenshot_CAF7FDFD-3A46-47DF-8261-63A779EA1FD1

File: HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift (hunk 6 above)
Comment (iOSKirill) [reply HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: Fixed

File: HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift (hunk 5 above)
Comment (iOSKirill) [reply HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: Fixed

Comment (iOSKirill) [reply HeartRateMonitorApp/HeartRateMonitorApp/HomeHealthModule/View/HomeHealthView.swift]: Fixed

File: HeartRateMonitorApp/HeartRateMonitorApp/CustomViews/CustomNavigationBar.swift (hunk 4 above)
Comment (iOSKirill) [reply HeartRateMonitorApp/HeartRateMonitorApp/CustomViews/CustomNavigationBar.swift]: Fixed

File: HeartRateMonitorApp/HeartRateMonitorApp/CustomViews/CustomNavigationBar.swift (hunk 3 above)
Comment (iOSKirill) [reply HeartRateMonitorApp/HeartRateMonitorApp/CustomViews/CustomNavigationBar.swift]: Fixed

**Review by ghost — APPROVED** [review]

**Pull request merged**